modules/contact-form/grunion-contact-form.php

```diff
@@ -1542,7 +1542,20 @@ static function parse( $attributes, $content ) {
 			$r .= "<form action='" . esc_url( $url ) . "' method='post' class='contact-form commentsblock'>\n";
 			$r .= $form->body;
 			$r .= "\t<p class='contact-submit'>\n";
-			$r .= "\t\t<input type='submit' value='" . esc_attr( $form->get_attribute( 'submit_button_text' ) ) . "' class='pushbutton-wide'/>\n";
+			
+			$submit_button_class = 'pushbutton-wide';
```

Comment:
This is a real nitpicky suggestion, but you could have avoided adding that line by just passing `pushbutton-wide` as a default to the `apply_filters` call down there.